src/assets/toolkit/styles/components/card.css

```diff
@@ -0,0 +1,127 @@
+/** @define Card */
+
+:root {
+  --Card-bg: var(--color-white);
+  --Card-border-color: var(--color-gray);
+  --Card-border-width: var(--border-width-md);
+  --Card-main-color: var(--link-color);
+  --Card-main-color-hover: var(--link-hover-color);
+  --Card-mainObject-size: calc(var(--ms5) * 1em);
+  --Card-meta-bg: color(var(--color-gray) l(+5%));
+  --Card-meta-color: #657BAE;
+  --Card-pad: var(--space-md);
+  --Card-pad-sm: var(--space-sm);
+}
+
+/**
+ * 1. Establish a border and background to cover all bases for offsetting from
+ *    containing elements.
+ * 2. Use Flexbox to allow main element to grow in cases where the element is
+ *    stretched.
+ * 3. Set up preferences for animation.
+ */
+
+.Card {
+  background-color: var(--Card-bg); /* 1 */
+  border: var(--Card-border-width) solid var(--Card-border-color); /* 1 */
+  display: flex; /* 2 */
+  flex-direction: column; /* 2 */
+  transform-origin: center; /* 3 */
+  transition: transform 0.2s easeInOutQuad; /* 3 */
+}
+
+/**
+ * Animation effect
+ */
+
+.Card:matches(:--enter) {
```

Comment:
I think this might be an artifact of when I had `:active` in here as well... will take a look.